README.md

```diff
@@ -1,31 +1,74 @@
 # Variable Assignment
 
 ## Learning Goals
-+ Describe how expression are used in programming
-+ Identify the different parts of an expression
++ Describe how expressions are used in programming
 + Identify the return value of an assignment expression
 + Describe the Variable Lookup Expression
 + Describe the Assignment Expression
 + Identify the constant expression
-+ Explain how the constant expression creates agreement
 
-## Outline
-Start with an open text editor with a blank ruby file - IDE Sandbox will work well. 
-
-+ Describe how Ruby parses through expressions
-  + All code is just text in a text file that lives on a computer
-  + When we execute our program, the ruby interpreter reads through the code, evaluates each expression, and gives it a return value. 
-  + An expression like "1 + 1", Ruby interprets this expression, and knows to return 2
-+ Talk about why to use variables
-  + If all of our programs used constant data like this, that wouldn't be very interesting. 
-  + If you think about an app like Twitter or Facebook - those are only interesting because of dynamic data. You can show different things to different people. 
-  + We can accomplish this using variables - a variable is something that can have many different values of the course of our program. 
-  + We can assign it using a single equals sign.
-  + number = 5, then do a bunch of math on the number. Then change number to 7 and run it again.
-+ Constants 
-  + Useful to call-out values that don't change during the course of our program
-  + Age in Dog Years - developer might change it from 7 to 4, but it won't change during our run time
-  + In Ruby, you'll typically see this to call attention to important stuff
-  + Maybe bring up magic numbers here 
+## Lesson
 
++ Hi guys, this is Ian from Flatiron School. In this video, we're going to look at variable assignment in Ruby.
++ By the end of this video, you should be able to:
+  + Describe how expression are used in programming
+  + Identify the return value of an assignment expression
+  + Describe the Variable Lookup Expression
+  + Describe the Assignment Expression
+  + Identify the constant expression
++ So I'm in the Learn IDE Sandbox here, with a ruby file called `app.rb`. This gives me a place to play around with code and to test stuff out.
++ Remember that all code is just text in a file that can be interpreted by a computer.
++ In Ruby, when we execute a file, the ruby interpreter is running through the file from top to bottom and looking at expressions.
++ An expression is just some combination of data or operators that will eventually evaluate to some value
```

Comment:
There might be too many new words here (not sure):
- data
- operator
- evaluate

Reply:
This could be "return some value" since you're talking about "return values" a few lines down.

Reply:
```suggestion
+ An expression is just some combination of code that will eventually return some value.
+ This combination could be data - such as a string of text or a number, or an operator, like the `+` sign. 
```

Reply:
@timothylevi - That's a good catch. What do you think of the above suggestion?